.eslintrc.js

```diff
@@ -1,15 +1,78 @@
 module.exports = {
-  root: true,
-  env: {
-    browser: true,
-    node: true,
-  },
-  extends: [
-    '@nuxtjs/eslint-config-typescript',
-    'plugin:prettier/recommended',
-    'plugin:nuxt/recommended',
-  ],
-  plugins: [],
-  // add your custom rules here
-  rules: {},
-}
+    root: true,
+    env: {
+        browser: true,
+        node: true
+    },
+    extends: [
+        "@nuxtjs/eslint-config-typescript",
+        // 'plugin:prettier/recommended',
+        "plugin:nuxt/recommended"
+    ],
+    plugins: [],
+    rules: {
+        semi: [ "warn", "always" ],
+        quotes: [ "warn", "double", { allowTemplateLiterals: true } ],
+        "max-len": [
+            "warn",
+            {
+                code: 180,
+                tabWidth: 4,
+                ignoreUrls: true
+            }
+        ],
+        indent: [
+            "warn",
+            4,
+            {
+                SwitchCase: 1
+            }
+        ],
+        "vue/max-attributes-per-line": [
+            "warn",
+            {
+                singleline: 1,
+                multiline: {
+                    max: 1,
+                    allowFirstLine: true
+                }
+            }
+        ],
+        "vue/html-closing-bracket-newline": [
+            "warn", {
+                singleline: "never",
+                multiline: "never"
+            }
+        ],
+        "template-tag-spacing": "warn",
+        "vue/singleline-html-element-content-newline": [ "warn", {
+            ignoreWhenNoAttributes: false,
+            ignoreWhenEmpty: true,
+            ignores: [ "pre", "textarea" ]
```

Comment:
In questo sono abituato a mettere gli header e gli span, probabilmente su un progetto piccolino non fa troppa differenza.. quindi se qualcun altro è abituato come parliamone, altrimenti mi adatto

Reply:
In linea di massima se un elemento non è strettamente necessario non lo metto, ma è indifferente, posso aggiungere gli span alla lista